FakeTrello/Models/TaggedUser.cs

```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class TaggedUser
```

Comment:
Instead of using a class like this, explore adding a property on your Card class for this.